src/shared/backend/services/cloudbuild.ts

```diff
@@ -88,6 +82,46 @@ export const createJob = async (params: JobStatusParams): Promise<Job> => {
   return data as Job;
 };
 
+function timeout(ms: number): Promise<void> {
```

Comment:
Consider adding this to `utils`, it may already be there

Reply:
should I add a utils.ts file?

Reply:
I thought there might already be a shared set of code, it may fit within there?

Reply:
okay, found it :)